service/matching/poller/history.go

```diff
@@ -55,65 +55,61 @@ type pollerHistory struct {
 // HistoryUpdatedFunc is a type for notifying applications when the poller history was updated
 type HistoryUpdatedFunc func()
 
-func newPollerHistory(historyUpdatedFunc HistoryUpdatedFunc) *pollerHistory {
+func NewPollerHistory(historyUpdatedFunc HistoryUpdatedFunc) *History {
 	opts := &cache.Options{
 		InitialCapacity: pollerHistoryInitSize,
 		TTL:             pollerHistoryTTL,
 		Pin:             false,
 		MaxCount:        pollerHistoryInitMaxSize,
 	}
 
-	return &pollerHistory{
+	return &History{
 		history:              cache.New(opts),
 		onHistoryUpdatedFunc: historyUpdatedFunc,
 	}
 }
 
-func (pollers *pollerHistory) updatePollerInfo(id pollerIdentity, info pollerInfo) {
+func (pollers *History) UpdatePollerInfo(id Identity, info Info) {
 	pollers.history.Put(id, &info)
 	if pollers.onHistoryUpdatedFunc != nil {
 		pollers.onHistoryUpdatedFunc()
 	}
 }
 
-func (pollers *pollerHistory) getPollerInfo(earliestAccessTime time.Time) []*types.PollerInfo {
+func (pollers *History) GetPollerInfo(earliestAccessTime time.Time) []*types.PollerInfo {
 	var result []*types.PollerInfo
 
 	ite := pollers.history.Iterator()
 	defer ite.Close()
 	for ite.HasNext() {
 		entry := ite.Next()
-		key := entry.Key().(pollerIdentity)
-		value := entry.Value().(*pollerInfo)
+		key := entry.Key().(Identity)
+		value := entry.Value().(*Info)
 		// TODO add IP, T1396795
 		lastAccessTime := entry.CreateTime()
 		if earliestAccessTime.Before(lastAccessTime) {
-			rps := _defaultTaskDispatchRPS
-			if value.ratePerSecond != nil {
-				rps = *value.ratePerSecond
-			}
 			result = append(result, &types.PollerInfo{
 				Identity:       string(key),
 				LastAccessTime: common.Int64Ptr(lastAccessTime.UnixNano()),
-				RatePerSecond:  rps,
+				RatePerSecond:  value.RatePerSecond,
```

Comment:
Previously this value was either `_defaultTaskDispatchRPS` or `value.ratePerSecond` if it's not nil.
Is there any place using this value that will be impacted by probably seeing more zeros?

Reply:
I updated the caller in this PR to let the caller pass the default value explicitly.
The reason is that I want the default value to be defined in one place and don't want to export that value to other packages. And now the default value is defined in `config/config.go`.